.babelrc

```diff
@@ -1,9 +1,14 @@
 {
-    "presets": ["es2015", "stage-0"],
-    "plugins": [
-        "syntax-async-functions",
-        "transform-regenerator",
```

Comment:
transform-regenerator is a part of es2015, and syntax-async-functions is already enabled by default.

We recommend using https://github.com/babel/babel-preset-env which covers es2015,16,17, and future updates.